xchange-binance/src/main/java/org/knowm/xchange/binance/service/BinanceTradeService.java

```diff
@@ -367,9 +363,52 @@ public OpenOrdersParams createOpenOrdersParams() {
                             .getExchangeSpecificParametersItem("recvWindow"),
                     getTimestamp())));
       }
-      return orders;
+
+      // Get fees
+      return addFees(orders);
     } catch (BinanceException e) {
       throw BinanceErrorAdapter.adapt(e);
     }
   }
+
+  private Collection<Order> addFees(Collection<Order> orders) {
+    List<BinanceTrade> binanceTrades = new ArrayList<>();
+    orders.stream()
+        .map(Order::getCurrencyPair)
+        .distinct() //Optimization to reduce overhead
+        .forEach(
+            c -> {
+              try {
+                binanceTrades.addAll(
+                    super.myTrades(
```

Comment:
this will not work for older orders

Reply:
Yeh older orders are also missing cummulativeQuoteQty from the API. Nothing we can do regarding older orders.

Reply:
look at my recent comment, if I do a lot of trades event orders from today may fail to get filled